keps/sig-testing/3041-node-conformance-and-features/README.md

```diff
@@ -0,0 +1,803 @@
+- Convert textual tags into the ginkgo v2 tags. This is optional, but not
+  required and can be handled outside of this KEP.
+
+## Proposal
+
+### Clarify the semantic of a `[Feature:]` label
+
+The current [feature definition is](https://github.com/kubernetes/community/blob/32a1c14d04ff78684d78b827ac7c49f70352d509/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests):
+
+> * `[Feature:.+]`: If a test has non-default requirements to run or targets
+>   some non-core functionality, and thus should not be run as part of the
+>   standard suite, it receives a `[Feature:.+]` label, e.g.
+>   `[Feature:Performance]` or `[Feature:Ingress]`. `[Feature:.+]` tests are not
+>   run in our core suites, instead running in custom suites. If a feature is
+>   experimental or alpha and is not enabled by default due to being incomplete
+>   or potentially subject to breaking changes, it does not block PR merges, and
+>   thus should run in some separate test suites owned by the feature owner(s)
+>   (see Continuous Integration below).
+
+Feature will indicate that things may NOT work on all k8s distros and/or node
+capabilities. This flag will not indicate the Feature Gate state.
+
+Proposed change:
+
+> * `[Feature:.+]`: If a test validates functionality that may only work
+>   outside the minimal conformant installation of Kubernetes, e.g. when
+>   specific node capabilities are enabled, having a certain addons available,
+>   like loadbalancer integration, or when test depends on the
+>   functionality of underlying components like container runtime, and thus may
+>   need to be skipped on certain environments, it receives a `[Feature:.+]`
+>   label, e.g. `[Feature:AppArmor]`. This label can also be applied to signify that
+>   test validates non-core functionality, like `[Feature:Ingress]`.
+>   `[Feature:.+]` tests are not run in a core suites, but typically can run together
+>   on the standard environment as many capabilities and functionality of underlying
+>   components are pre-enabled on the standard environment. Use `[Environment:Foo]` to
+>   signify the specific environment configuration needed to run the test.
+>   For example if test requires GPU to be configured, but tests the non-optional
+>   feature like a device plugin.
+>   `[Feature:+]` label should not be confused with `[FeatureGate:]` label.
+
+And another two:
+
+> * `[FeatureGate:.+]`: If a test only works when the certain feature gate is
+>   enabled it receives a `[FeatureGate:.+]` label. `[FeatureGate:.+]` tests
+>   must also be marked with the status of this feature gate: `[Alpha]`,
+>   `[Beta]`, `[Stable]`, `[Deprecated]`. This label helps to skip tests that
+>   should not work on specific k8s distributive that has a certain feature gate
+>   disabled. This label has to be removed when feature gate value is "locked" or
+>   removed.
+
+> * `[Environment:.+]`: If test requires non-standard environment
+>   (different from standard OSS test machines) to run it
```

Comment:
this tag is mostly to filter tests out. Like run everything that doesn't require non-standard environment. Instead of filtering out `[Feature:Performance]`